scalafix-cli/src/main/scala/scalafix/cli/CliRunner.scala

```diff
@@ -397,20 +415,34 @@ object CliRunner {
           .notOk
     }
 
+    val diffDisable: Option[DiffDisable] = {
+      if (cli.diff || cli.diffBranch.nonEmpty) {
+        val baseBranch = cli.diffBranch.getOrElse("master")
+        Some(DiffDisable(common.workingPath.toNIO, baseBranch))
```

Comment:
What happens when `DiffDisable` fails? For example a non-existent branch is used. We can convert `diffDisable` into a `Configured[Option[DiffDisable]]` and `(resolvedPathMatcher |@| diffDisable).map`